src/moj/filters/all.js

```diff
@@ -60,8 +73,17 @@ module.exports = function () {
     }
   }
 
-  /* ------------------------------------------------------------------
-    keep the following line to return your filters to the app
-  ------------------------------------------------------------------ */
-  return filters
+  /**
+   * Returns an object used to store the methods registered as a
+   * 'filter' (of the same name) within nunjucks. You can override
+   * gov.uk core filters by creating filter methods of the same name.
+   */
+  return {
+    date,
+    mojDate
```

Comment:
Could the fact we are not exporting the other functions cause anyone a problem if they are using them directly in prototypes? Or are these functions suplicates of ones already available within the prototype kit?